pkg/sql/parser/sql.y

```diff
@@ -802,7 +802,7 @@ func newNameFromStr(s string) *tree.Name {
 %type <tree.IsolationLevel> iso_level
 %type <tree.UserPriority> user_priority
 
-%type <tree.TableDefs> opt_table_elem_list table_elem_list
+%type <tree.TableDefs> opt_table_elem_list table_elem_list ctas_opt_col_list ctas_table_elem_list
```

Comment:
here/thoughout: for someone else coming into this code or trying to read the diagrams, I think "ctas" could be more descriptive. maybe "create_as" ?